packages/chain-adapters/src/evm/avalanche/AvalancheChainAdapter.test.ts

```diff
@@ -0,0 +1,706 @@
+// Allow explicit any since this is a test file
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/**
+ * Test AvalancheChainAdapter
+ * @group unit
+ */
+import { ETHSignMessage, ETHSignTx, ETHWallet } from '@shapeshiftoss/hdwallet-core'
+import { NativeAdapterArgs, NativeHDWallet } from '@shapeshiftoss/hdwallet-native'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import unchained from '@shapeshiftoss/unchained-client'
+import { merge } from 'lodash'
+import { numberToHex } from 'web3-utils'
+
+import {
+  BuildSendTxInput,
+  SignMessageInput,
+  SignTxInput,
+  ValidAddressResultType
+} from '../../types'
+import { bn } from '../../utils/bignumber'
+import { ChainAdapterArgs } from '../EVMBaseAdapter'
+import * as avalanche from './AvalancheChainAdapter'
+
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+const EOA_ADDRESS = '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045'
+const ENS_NAME = 'vitalik.eth'
+const VALID_CHAIN_ID = 'eip155:43114'
+
+const testMnemonic = 'alcohol woman abuse must during monitor noble actual mixed trade anger aisle'
+
+const getWallet = async (): Promise<ETHWallet> => {
+  const nativeAdapterArgs: NativeAdapterArgs = {
+    mnemonic: testMnemonic,
+    deviceId: 'test'
+  }
+  const wallet = new NativeHDWallet(nativeAdapterArgs)
+  await wallet.initialize()
+
+  return wallet
+}
+
+jest.mock('axios', () => ({
+  get: jest.fn(() =>
+    Promise.resolve({
+      data: {
+        result: [
+          {
+            source: 'MEDIAN',
+            timestamp: 1639978534,
+            instant: 55477500000,
+            fast: 50180000000,
+            standard: 45000000000,
+            low: 41000000000
+          }
+        ]
+      }
+    })
+  )
+}))
+
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('AvalancheChainAdapter', () => {
```

Comment:
unchained client does not currently support avalanche chain id, skipping tests until updated with support